doc/calls_and_callbacks.md

```diff
@@ -1 +0,0 @@
-![psen_scan_v2_callbacks](api_sequence_diag.png)
```

Comment:
what is this? where does it come from, where does it go?

Reply:
You are referring to a line that was taken out, is this outdated or did you mean something else?